public/static/docs/api-reference/get_url.md

```diff
@@ -82,10 +83,13 @@ resource_url = dvc.api.get_url(
     'get-started/data.xml',
     repo='https://github.com/iterative/dataset-registry'
     )
+
 print(resource_url)
 ```
 
----
+It prints
+`https://remote.dvc.org/dataset-registry/a3/04afb96060aad90176268345e10355` if
+we run it:
 
 ```dvc
 $ python script.py
```

Comment:
This is repetitive. Maybe just something like "Prints the URL string value if we run it from terminal:" ?

Reply:
Again, it's easier to read. Repetition not a big problem. Code snippet just illustrates the code (and gives a command to run). I would keep it extra explicit. Up yo you again. Don't have a strong opinion

Reply:
> Repetition not a big problem.

Yeah but why do it if it's not needed? Just keeping the top paragraph and end it in "if we run it.", and removing the console block code should be enough I think.

> The original one I did with the `---` hr was also kind of obvious, like in the basic usage comment 🙂

Reply:
> The original one I did with the --- hr was also kind of obvious, like in the basic usage comment 🙂 and I like how it looked rendered

hr looks like an overkill ... it's a split of the whole document, not very suitable to split example and it's output

Reply:
OK, true.